app/models/manageiq/providers/google/inventory/parser/cloud_manager.rb

```diff
@@ -318,9 +318,11 @@ def parse_compute_metadata_ssh_keys(metadata)
     # Find the sshKeys property in the instance metadata
     metadata_ssh_keys = parse_compute_metadata(metadata, "sshKeys")
 
-    metadata_ssh_keys.to_s.split("\n").each do |ssh_key|
+    metadata_ssh_keys.to_s.split("\n").reject(&:blank?).each do |ssh_key|
       # Google returns the key in the form username:public_key
       name, public_key = ssh_key.split(":", 2)
+      next if public_key.nil?
```

Comment:
Should we make that `public_key.blank?` just to be sure?

Reply:
:+1: good point

Reply:
Done